discovery-infra/test_infra/utils/utils.py

```diff
@@ -685,7 +685,14 @@ def get_kubeconfig_path(cluster_name: str) -> str:
     return kubeconfig_path
 
 
-def get_openshift_version(default=consts.DEFAULT_OPENSHIFT_VERSION):
+def get_default_openshift_version() -> str:
+    stdout, _, _ = run_command(
+        f"jq -r 'to_entries | .[] | select(.value.default) | .key' assisted-service/data/default_ocp_versions.json", shell=True
```

Comment:
if you run test-infra on remote-url, without spawning minikube and deploying service, you will not get this fille.
I think you should validate it before.
Why to use jq? You are using python , just json.load it

Reply:
That's true. So maybe we should use the client versions API instead.